test/library/standard/IO/readBinary/multiloc/binaryString.chpl

```diff
@@ -4,7 +4,7 @@ on Locales[0] {
     var reader = openReader("./input.bin");
 
     on Locales[1] {
-        var s: string = createStringWithNewBuffer("");
+        var s: string = string.createCopyingBuffer("");
```

Comment:
Same as above, I think this overload shouldn't exist. This is just `var s = "";`

Reply:
Taking another look at the implementations, I think you did the right thing. Is this test passing?

Reply:
> The test passes as-is

This is somewhat disconcerting. I wonder if there's a `string` to `c_string` coercion. I doubt that that exists.. Not necessarily asking you to figure out what's going on in this particular test, but this doesn't work with this PR, right?:

```chapel
var s1 = "first string";
var s2 = string.createCopyingBuffer(s1);  // should be unresolved
```

Reply:
> but this doesn't work with this PR, right?

Yes, it fails to resolve.

Reply:
The test's behavior is strange, but that's reassuring.